olm-catalog/serverless-operator/project.yaml

```diff
@@ -61,7 +61,7 @@ dependencies:
         openjdk_17: registry.access.redhat.com/ubi8/openjdk-17@sha256:b00f687d913b8d1e027f7eabd6765de6c8d469629bef9550f10dbf207af24fe5
         openjdk_21: registry.access.redhat.com/ubi8/openjdk-21@sha256:4f35566977c35306a8f2102841ceb7fa10a6d9ac47c079131caed5655140f9b2
         python-39: registry.access.redhat.com/ubi8/python-39@sha256:cf0af1732c483d4e6ba708f9f4d5541cb43c98c3c67c604c23b0e55897eebe41
-    operator: 1.11.0
+    operator: 1.13.0
```

Comment:
Can we go w/ the "fresh" `1.13.3`:
https://github.com/knative/operator/tree/knative-v1.13.3

CC @pierDipi

Reply:
+1

Reply:
@ReToCode  ^

Reply:
Yes let's get the latest if possible.

Reply:
This is just for the yaml (not code). I did run it with that, no changes to the 1.13.0. I can still update, but doesn't really change anything.